.github/.release-please-manifest.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need this file since it's empty?

Reply:
Yeah, it is filled in by the release please bot as it releases new versions. I believe the convention of starting the file with a `.` is to remind people it's generated.